src/plugins/simulator/fs9/simulatorfs9.h

```diff
@@ -103,6 +103,8 @@ namespace BlackSimPlugin::Fs9
         QSharedPointer<CLobbyClient> m_lobbyClient;
 
         static constexpr int SkipUpdateCyclesForCockpit = 10;
+
+        bool dontOverwrite(const BlackMisc::PhysicalQuantities::CFrequency &from, const BlackMisc::PhysicalQuantities::CFrequency &to);
```

Comment:
As this method is only used inside ``CSimulatorFs9::updateOwnAircraftFromSimulator``, we could continue using it as a local lambda.

Reply:
That's right, I planned to use this function elsewhere to check for 132,200 and 131,200. Change is revoked